ably-ios/ARTRealtime.m

```diff
@@ -142,6 +140,18 @@ - (ARTLog *)getLogger {
     return _rest.logger;
 }
 
+- (ARTClientOptions *)getClientOptions {
+    return _rest.options;
+}
+
+- (NSString *)getClientId {
```

Comment:
Can this not simply be `clientId()`?  I know we discussed this before, but since we did I've lost track of where we got to, sorry :/